### PR TITLE
i18n(es): Update `from-gatsby`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description (required)

Updated `from-gatsby`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
+    - `Gatsby` se convierte en `Astro` 🚀
+
+    ```astro title="src/layouts/Layout.astro" "<slot />" "class" "Astro" "astro.build"
+    ---
+    ---
+    <div class="global-wrapper" data-is-root-path={isRootPath}>
+      <header class="global-header">{header}</header>
+      <main><slot /></main>
+      <footer>
+        © {new Date().getFullYear()}, Built with
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:448]:
```suggestion
        © {new Date().getFullYear()}, Construido con
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 2]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
+        <title>Astro</title>
+      </head>
+      <body>
+        <div class="global-wrapper" data-is-root-path={isRootPath}>
+          <header class="global-header">{header}</header>
+          <main>
+            <slot />
+          </main>
+          <footer>
+            &#169; {new Date().getFullYear()}, Built with
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:474]:
```suggestion
            &#169; {new Date().getFullYear()}, Construido con
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 3]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
+        <div class="global-wrapper">
+          <header class="global-header">
+            { pathname === "/" 
+            ? 
+              <h1 class="main-heading">
+              <a href="/">{title}</a>
+              </h1>
+            : 
+              <h1 class="main-heading">
+              <a class="header-link-home" href="/">Home</a>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:516]:
```suggestion
              <a class="header-link-home" href="/">Inicio</a>
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 4]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
+              </h1>
+            }  
+          </header>
+          <main>
+            <slot />
+          </main>
+          <footer>
+            &#169; {new Date().getFullYear()}, Built with
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:524]:
```suggestion
            &#169; {new Date().getFullYear()}, Construido con
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 5]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
-  ```
-  :::tip
-  Puedes [obtener la ruta de la página actual utilizando `Astro.url`](/es/reference/api-reference/#astrourl).
-  :::
+    ```astro title="src/pages/index.astro"
+    ---
+    import Layout from '../layouts/Layout.astro';
+    const pagePathname = Astro.url.pathname
+    ---
+    <Layout title="Home Page" pathname={pagePathname}>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:540]:
```suggestion
    <Layout title="Página de inicio" pathname={pagePathname}>
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 6]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
-  </Layout>
-  ```
-
-  Sólo deberías ver un enlace a "Home" cuando visites la página About.
+    ```astro title="src/pages/about.astro"
+    ---
+    import Layout from '../layouts/Layout.astro';
+    const pagePathname = Astro.url.pathname
+    ---
+    <Layout title="About" pathname={pagePathname}>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:555]:
```suggestion
    <Layout title="Acerca de" pathname={pagePathname}>
```

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 7]
```diff
@@ -394,170 +394,170 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
...
+        <p>About</p>
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:556]:
```suggestion
        <p>Acerca de</p>
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Good job as always 🙌

**Pull request merged**